bin/local-env/docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   wordpress:
     image: wordpress
     ports:
-      - '127.0.0.1:8899:80'
+      - '8899:80'
```

Comment:
@swissspidy Can you explain why this IP was put here in the first place?

Reply:
Basically to prevent exactly what this PR does (access from other devices in the network) :-)